scalyr_agent/tests/copying_manager_test.py

```diff
@@ -1048,10 +1048,11 @@ class _time_mock(object):
             # This dummy 'time()' should be called on new copying thread iteration
             # to emulate huge gap between last request.
             def time(_self):
-                res = orig_time.time() + self._manager._CopyingManager__config.max_retry_time
-                # restore original 'time' module
-                copying_manager.time = orig_time
-                return res
+                try:
+                    return orig_time.time() + self._manager._CopyingManager__config.max_retry_time
+                finally:
```

Comment:
Move the finally to after line 1067... essentially, the end of the test case.

Essentially, I like to see the `try/finally` be at the same level as where you are overriding the `time` value.

Reply:
Oh, how could I forget about this. 
Done.